packages/react-components/react-charts-preview/src/components/CommonComponents/CartesianChart.tsx

```diff
@@ -24,6 +24,10 @@ import {
 } from '../../utilities/index';
 import { SVGTooltipText } from '../../utilities/SVGTooltipText';
 import PopoverComponent from './Popover';
+import { useFocusableGroup } from '@fluentui/react-tabster';
```

Comment:
Have you created a table for behavior in v8 and in v9. Any differences observed.

Reply:
Will create that